docset/windows/applocker/New-AppLockerPolicy.md

```diff
@@ -81,6 +81,8 @@ If only path information is available for a file, then the file is skipped becau
 If the *IgnoreMissingFileInformation* parameter is not specified when file information is missing, then the cmdlet exits because it cannot create the specified rule type.
 After the new AppLocker policy is created, the AppLocker policy of the specified Group Policy Object (GPO) is set.
 The existing AppLocker policy in the specified GPO will be overwritten.
+If *where {$_.path -like '*.exe'}* is not specified it could cause the workstations to enable rule enforcement on DLL files and cause the workstation to not trust Windows DLL files
```

Comment:
The word _Workstation_ is not used in these articles, instead _computer_ it is, can we use that word?
Also, I think this paragraph should be before the _After the new AppLocker policy_ paragraph